### PR TITLE
Add support for logsets (aggregated logs from multiple hosts)

### PR DESCRIPTION
**Pull request by pst**
- default[:logentries][:logs] are followed per host
- default[:logentries][:logsets] are aggregated in a logset
  named after the first part of the hostname. E.g. for
  lxc-dev-muycqh the logset would be called lxc

### CODE REVIEW

File: logentries/recipes/default.rb [hunk 1]
```diff
@@ -26,28 +26,44 @@
...
+package "logentries-daemon"
+
+# Some logs go into a logset, some go into the host. The config template
+# is required for the ones that go into the logset
+
+directory "/etc/le" do
+  owner 'root'
+  group 'root'
+  mode '0440'
+  action :create
```
Comment (sfriesel) [inline logentries/recipes/default.rb]: not needed, `:create` is default

File: logentries/recipes/default.rb [hunk 2]
```diff
@@ -26,28 +26,44 @@
...
 package "logentries"
+package "logentries-daemon"
+
+# Some logs go into a logset, some go into the host. The config template
+# is required for the ones that go into the logset
+
+directory "/etc/le" do
+  owner 'root'
+  group 'root'
+  mode '0440'
```
Comment (sfriesel) [inline logentries/recipes/default.rb]: that means even root can't access the directory? I would use `0755`

File: logentries/recipes/default.rb [hunk 3]
```diff
@@ -26,28 +26,44 @@
...
+  action :create
+end
+
+template "/etc/le/config" do
+  source "config.erb"
+  mode '0440'
```
Comment (sfriesel) [inline logentries/recipes/default.rb]: `0600`?

**Comment by pst** [comment]
Comments fixed

File: logentries/attributes/default.rb [hunk 4]
```diff
@@ -1,4 +1,5 @@
-default[:logentries][:logs][:syslog] = { "filename" => '/var/log/syslog', 'name' => 'syslog', 'type' => 'syslog' }
 default[:logentries][:logs][:chefclient] = { "filename" => '/var/log/chef/client.log', 'name' => 'chef-client.log', 'type' => 'syslog' }
 default[:logentries][:logs][:cloudinit] = { "filename" => '/var/log/cloud-init.log', 'name' => 'cloud-init.log', 'type' => 'syslog' }
 default[:logentries][:logs][:kern] = { "filename" => '/var/log/kern.log', 'name' => 'kern.log', 'type' => 'syslog' }
+
+default[:logentries][:logsets][:syslog] = { "filename" => '/var/log/syslog', 'name' => 'syslog', 'type' => 'syslog' }
```
Comment (TooAngel) [inline logentries/attributes/default.rb]: I would also remove the type parameter if it isn't used anymore

**Comment by pst** [comment]
Removed type from attributes.

**Pull request merged**